setup.py

```diff
@@ -296,7 +296,9 @@ def add_unique(l: List[str], vals: Union[str, List[str]]) -> None:
         if not found_pybind11():
             add_unique(setup_reqs, "pybind11")
         add_unique(install_reqs, ["jax", "flax"])
-        add_unique(test_reqs, ["numpy", "praxis"])
+        # Installing with SKIP_HEAD_INSTALLS=true skips jax/fiddle. jax is covered
+        #   in install_reqs, so fiddle is covered in test_reqs
+        add_unique(test_reqs, ["numpy", "praxis @ git+https://github.com/google/praxis.git", "fiddle"])
```

Comment:
I think the `SKIP_HEAD_INSTALLS` method should be considered an advanced technique and we shouldn't put too much effort in TE to support it. If the user knows what they're doing, they should already have a Praxis installation that's new enough to support it.
```suggestion
        add_unique(test_reqs, ["numpy", "praxis", "fiddle"])
```